File: core/Core/Models/PaymentTemplate.cs
using System;
using System.Collections.Generic;

namespace Core.Models;

public partial class PaymentTemplate
{
    public int TemplateId { get; set; }

    public required int CustomerId { get; set; }

    public required string Name { get; set; }

    public required string TargetIban { get; set; }

    public decimal? DefaultAmount { get; set; }

    public required Customer Customer { get; set; }
}

File: analytic/domain/payment_template/payment_template.py
class PaymentTemplate:
    def __init__(self, template_id: int, customer_id: int, name: str, target_iban: str, default_amount: str):
        self.default_amount = default_amount
        self.target_iban = target_iban
        self.name = name
        self.customer_id = customer_id
        self.template_id = template_id